server/command-line/install.ts

```diff
@@ -47,9 +47,20 @@ program
 				.readFile(path.join(packageName.substring("file:".length), "package.json"), "utf-8")
 				.then((data) => JSON.parse(data) as typeof packageJson);
 		} else {
-			const split = packageName.split("@");
-			packageName = split[0];
-			const packageVersion = split[1] || "latest";
+			let packageVersion = "latest";
+
+			if (packageName.startsWith("@")) {
+				const atIndex = packageName.lastIndexOf("@");
+
+				if (atIndex > 0) {
+					packageVersion = packageName.slice(atIndex + 1) || "latest";
+					packageName = packageName.slice(0, atIndex);
+				}
+			} else {
+				const split = packageName.split("@");
+				packageName = split[0];
+				packageVersion = split[1] || "latest";
+			}
```

Comment:
This is more complicated than need be I think...
You don't have to special case anything here, just start looking at the 2nd char

```ts
let packageVersion = "latest"
const atIndex = packageName.lastIndexOf("@", 1); // make sure scoped @org/something works
if(atIndex != -1) {
    packageName = packageName.slice(0,atIndex);
    packageVersion = packageName.slice(atIndex+1);
}
```
Or such would work and remove most of the conditionals

Reply:
(copy pasta leftover from your code fixed... mea culpa)

Reply:
Thanks! I guess I was overthinking this a little, I just had to change packageName and packageVersion around, so that the sliced packageName doesn't get used for the Version